garnett/gsdhoomdfilewriter.py

```diff
@@ -10,74 +10,12 @@
 import gsd
 import gsd.hoomd
 import logging
-import numpy as np
-
-from .shapes import SphereShape, ConvexPolyhedronShape, ConvexSpheropolyhedronShape, \
-    PolygonShape, SpheropolygonShape, EllipsoidShape
-from .errors import GSDShapeError
 
 from .trajectory import PARTICLE_PROPERTIES
 
 logger = logging.getLogger(__name__)
 
 
-def _write_shape_definitions(snap, shapedefs):
```

Comment:
Yes, that was dropped in favor of writing `type_shapes` in #127. I mentioned it in a note on that PR description. Otherwise we could be writing HPMC state into a DEM simulation, for example. I added a specific note in the changelog.

Reply:
Thanks for the reminder. It totally slipped my mind.